__init__.py

```diff
@@ -15,7 +15,47 @@ class MarkExperience(Event):
 
 
 MODIFIER_REGEX = "[+-]?[0,1,2,3]"
-STAT_REGEX = f"(?:(?:!cool|!tough|!sharp|!charm|!weird) {MODIFIER_REGEX})"
+GAME_STATS = {"motw": ["cool", "tough", "sharp", "charm", "weird"],
+              "pbtastartrek": ["aggressive", "bold", "talk", "tech"]}
+STAT_REGEXES = {}
+
+
+@match_event(OpsdroidStarted)
+@memory_in_event_room
```

Comment:
This isn't going to work, as `OpsdroidStarted` doesn't have a room. You will have to iterate over all the joined rooms?

Reply:
c.f. https://github.com/opsdroid/opsdroid/issues/1732

Reply:
For now I suggest you iterate over `opsdroid.get_connector('matrix').connection.rooms.values()`.